ext/net/ops.rs

```diff
@@ -608,6 +611,145 @@ where
   net_listen_udp::<NP>(state, addr, reuse_address, loopback)
 }
 
+#[cfg(unix)]
+#[op2(async, stack_trace)]
+#[serde]
+pub async fn op_net_connect_vsock<NP>(
+  state: Rc<RefCell<OpState>>,
+  #[smi] cid: u32,
+  #[smi] port: u32,
+) -> Result<(ResourceId, (u32, u32), (u32, u32)), NetError>
+where
+  NP: NetPermissions + 'static,
+{
+  use tokio_vsock::VsockAddr;
+  use tokio_vsock::VsockStream;
+
+  super::check_unstable(
+    &state.borrow(),
+    r#"Deno.connect({ transport: "vsock" })"#,
+  );
```

Comment:
Use a granular unstable flag here `--unstable-vsock` see the `UnstableGranularFlag` struct.